pkg/notifications/notifications_integration_test.go

```diff
@@ -63,6 +63,15 @@ func TestNotificationMutationDestination(t *testing.T) {
 	require.NotNil(t, getDestinationResult)
 	assert.Equal(t, 1, getDestinationResult.TotalCount)
 
+	// Test: Get Destination by name
+	filtersByName := ai.AiNotificationsDestinationFilter{
```

Comment:
Could you please extract this into a separate test? 
With the current setup, if the destination filtering logic would break, it would break the test called `TestNotificationMutationDestination`, which is quite misleading

Reply:
done!